communities/massmesh/meshradio/aerohive_hiveap-330/files/etc/config/wireless

```diff
@@ -0,0 +1,34 @@
+config wifi-device 'radio0'
+        option type 'mac80211'
+        option path 'ffe09000.pcie/pci9000:00/9000:00:00.0/9000:01:00.0'
+        option htmode 'HT20'
+        option country 'US'
+        option cell_density '0'
+        option hwmode '11g'
+        option channel 'auto'
+
+config wifi-iface 'default_radio0'
+        option device 'radio0'
+        option ifname 'radio011s'
```

Comment:
Is this a custom name? It looks like radio2 is the mesh one so this might be confusing

Reply:
Good catch!